client/src/Components/CardContainer/index.js

```diff
@@ -9,15 +9,24 @@ const { Title } = Typography;
 
 const CardContainer = ({ data, searchText }) => (
   <div>
-    <Title level={3} id="container-title">
-      {searchText ? `نتائج البحث عن: ${searchText}` : `أفضل رياض الأطفال`}
-    </Title>
     <div className="container-cards">
+      <Title level={3} id="container-title">
+        {searchText ? `نتائج البحث عن: ${searchText}` : `أفضل رياض الأطفال`}
+      </Title>
       <ul>
         {data.length !== 0 ? (
           data.map((item) => (
             <li key={item.id}>
-              <Card {...item} />
+              <Card
+                id={item.id}
+                title={item.kindergarten_name}
+                rating={parseFloat(item.rating_average)}
+                reviewersNo={parseFloat(item.rating_count)}
+                location={item.location_sub}
```

Comment:
```suggestion
                mainLocation={item.location_main}
                subLocation={item.location_sub}
```
And at the `Card` component edit these lines 
```js
const CardComponent = ({
  id,
  title,
  rating,
  reviewersNo,
  subLocation,
  mainLocation,
  minPrice,
  maxPrice,
  cover,
}) => {
```

```jsx
<h3> {`${mainLocation} - ${subLocation}`}</h3>
```

```js
CardComponent.propTypes = {
  id: PropTypes.number.isRequired,
  title: PropTypes.string.isRequired,
  rating: PropTypes.number.isRequired,
  reviewersNo: PropTypes.number.isRequired,
  subLocation: PropTypes.string.isRequired,
  mainLocation: PropTypes.string.isRequired,
  minPrice: PropTypes.number.isRequired,
  maxPrice: PropTypes.number.isRequired,
  cover: PropTypes.string.isRequired,
};
```